content/sigs/pipeline-authoring/index.adoc

```diff
@@ -85,6 +85,7 @@ Some examples of such posts/guides:
 
 Meetings are held every Friday at 9:00 AM PST. You can find more details by visiting the link:/event-calendar/[community calendar]
 Prior to each meeting a link to the meeting and meeting notes are provided in the link:https://gitter.im/jenkinsci/pipeline-authoring-sig[Jenkins Pipeline-Authoring Gitter]
+The last Friday of every month, the meeting is dedicated to link:https://www.jenkins.io/project/roadmap/[roadmap] discussions and statuses.
```

Comment:
Prefer to use a site local link rather than an absolute link.
```suggestion
The last Friday of every month, the meeting is dedicated to link:/project/roadmap/[roadmap] discussions and statuses.
```